scripts/xMetric/addTransactor.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "hardhat";
+import fs from "fs";
+
+async function main() {
+  const newTrasactor = "";
+
+  const network = process.env.HARDHAT_NETWORK;
+  console.log("adding transactor on: ", network);
+
+  const data = await fs.readFileSync("./deployments/" + network + "/Xmetric.json", "utf8");
+  const object = JSON.parse(data);
+  const address = object.address;
```

Comment:
the problem I see with this is if we are using forge to deploy I'm not sure exaclty how it format is compared to hardhat-deploy which adds an address

Reply:
yeah let's use hardhat to deploy for now - I really find a ton of value in these json files